### DISCUSSION TITLE
account blocked just after Pro purchase. All production apps blocked!

### DISCUSSION THREAD

**Discussion by kombos** [vercel/community#2559] [Help] [answered]
### Summary

facing a high level priority issue where we purchased the pro subscription and deployed all our production apps/sites to it. The very next day it started saying overdue instantly when we changed the owner of the pro plan and now saying as blocked. All our production apps are down and there is no contact number or chat provided to resolve such priority 1 issues!! please provide assistance asap. 

account: team_Vo3IyjVGXOkUV09JuSgLPO8B

**Comment by swarnava** [comment, marked as answer]
Hi,

We have responded to you on a**n@*******.one email
